content/02.introduction.md

```diff
@@ -10,66 +10,62 @@ One week later, 4593 confirmed cases had been identified, spanning not only Asia
 Coronavirus genomes are comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. 
 There are additional accessory genes that are present depending on the strain. 
-The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except Gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA [@doi:10.1002/jmv.25681]. 
+The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA [@doi:10.1002/jmv.25681]. 
 
 Replication initiates with the binding of the virus to a host cell via the S protein. 
 The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. 
 The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
 The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].  
 
-#### Transmission:
+#### 2.2.d Transmission
 In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes). 
-While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it is suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies[@doi:10.3390/ijerph15122757].
+While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it has been suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies [@doi:10.3390/ijerph15122757].
+
+### 2.3 SARS-CoV-2: Origin, Phylogenetics, and Evolution
 
-### Origin of SARS-CoV-2 
-[MERS and SARS? Where to put this?]
-[Specifically about SARS-CoV-2/COVID19?]
-The origin of the SARS-CoV-19 virus is not yet fully understood.
-Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-19 is unlikely to have originated from a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
+#### 2.3.a Origin of SARS-CoV-2
+The origin of the SARS-CoV-2 virus is not yet fully understood.
```

Comment:
Does this mean the species that it leapt from to humans? I guess at this point I view it as safe to say the origin is zoonotic transmission but the previous host or hosts remains unknown.

Reply:
How about:
```suggestion
The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
```
(I think Ronnie wrote this, but @RLordan added related text elsewhere if I recall correctly, so he might also have an opinion!)

Reply:
I envisioned this section to talk about the phylogenetic analyses that have been done to examine the intermediate host. I think there are some predictions but nothing conclusive from what I can see.

Reply:
@ajlee21 does this way look OK to you? Or did you mean something else? Also, sorry that I forgot you were the one who wrote this, I should have checked the blame!